ng2-components/ng2-activiti-form/src/components/widgets/people/people.widget.html

```diff
@@ -1,13 +1,13 @@
 <div class="adf-people-widget {{field.className}}"
      [class.is-dirty]="value"
-     [class.adf-invalid]="!field.isValid" [class.adf-readonly]="field.readOnly" id="typehead-div" *ngIf="field.isVisible">
+     [class.adf-invalid]="!field.isValid" [class.adf-readonly]="field.readOnly" id="people-div" *ngIf="field.isVisible">
```

Comment:
Can we have class name without "div" suffix, as it hardcodes the notion of the div element decorated.